lib/active_support_type_extensions/array.rb

```diff
@@ -9,7 +9,7 @@ def serializable?(_)
     def cast_value(value)
       case value
       when String
-        value.split(",")
+        cast_value(value.split(/,|\|/))
```

Comment:
Nitpick: this escaping hurt my brain a bit. I found this slightly easier to read:

```suggestion
        cast_value(value.split(/[,|]/))
```

Reply:
Yes, I had that thought too at some point, you’re right :+1: :sweat_smile: